apps/prairielearn/src/components/QuestionsTable.html.ts

```diff
@@ -45,6 +46,7 @@ export function QuestionsTable({
   urlPrefix: string;
   plainUrlPrefix: string;
   __csrf_token: string;
+  findQidMode?: boolean;
```

Comment:
The convention with this table is to name options by what they do (`showSharingSets`, `showAddQuestionButton`, etc.). I'd rather do the same here (maybe `showCheckboxes`).

Reply:
Updated name to `showCheckboxes` in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).